src/content/guides/development.md

```diff
@@ -274,7 +274,7 @@ __package.json__
     "scripts": {
       "test": "echo \"Error: no test specified\" && exit 1",
       "watch": "webpack --watch",
-+     "start": "webpack-dev-server --open",
++     "start": "webpack serve --open",
```

Comment:
I got error `error: option '--open <value>' argument missing`, am I missing something?

Here's the versions:

```
    "webpack": "^5.2.0",
    "webpack-cli": "^4.1.0",
    "webpack-dev-server": "^3.11.0",
```

Reply:
Bug, I will fix it in the near future, but it will work